optuna_dashboard/static/components/GraphParallelCoordinate.tsx

```diff
@@ -1,17 +1,74 @@
 import * as plotly from "plotly.js-dist"
-import React, { FC, useEffect } from "react"
+import React, { FC, useEffect, useState } from "react"
+import {
+  Grid,
+  FormControl,
+  FormLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core"
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles"
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    title: {
+      margin: "1em 0",
+    },
+    formControl: {
+      marginBottom: theme.spacing(2),
+      marginRight: theme.spacing(5),
+    },
+  })
+)
 
 const plotDomId = "graph-parallel-coordinate"
 
 export const GraphParallelCoordinate: FC<{
   study: StudyDetail | null
 }> = ({ study = null }) => {
+  const classes = useStyles()
+  const [objectiveId, setObjectiveId] = useState<number>(0)
+
+  const handleObjectiveChange = (
+    event: React.ChangeEvent<{ value: unknown }>
+  ) => {
+    setObjectiveId(event.target.value as number)
+  }
+
   useEffect(() => {
     if (study !== null) {
-      plotCoordinate(study, 0) // TODO(c-bata): Support multi-objective studies.
+      plotCoordinate(study, objectiveId)
     }
-  }, [study])
-  return <div id={plotDomId} />
+  }, [study, objectiveId])
+
+  return (
+    <Grid container direction="row">
+      <Grid item xs={3}>
+        <Grid container direction="column">
+          <Typography variant="h6" className={classes.title}>
+            Parallel cooridinate
+          </Typography>
+          {study !== null && study.directions.length !== 1 ? (
+            <FormControl component="fieldset" className={classes.formControl}>
+              <FormLabel component="legend">Objective ID:</FormLabel>
+              <Select value={objectiveId} onChange={handleObjectiveChange}>
+                {study.directions.map((d, i) => (
+                  <MenuItem value={i} key={i}>
+                    {i}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+          ) : null}
+        </Grid>
+      </Grid>
+
+      <Grid item xs={9}>
+        <div id={plotDomId} />
+      </Grid>
+    </Grid>
+  )
 }
 
 const plotCoordinate = (study: StudyDetail, objectiveId: number) => {
```

Comment:
How about disabling Plotly's chart title (e.g. "Parallel Coordinate", "Empirical Distribution Function plot", and "Hyperparameter Importance") like History and Slice plots? It looks duplicated.

![study-single](https://user-images.githubusercontent.com/5564044/121005952-4f709980-c7cb-11eb-9b8f-c693f328b757.png)

Reply:
Just disabled plotly titles in these three charts. As for adding selections to filter trial states, I think, as you said, we can leave that to another PR.

Reply:
OK! Thank you for the update.